obspy/core/tests/test_waveform_plugins.py

```diff
@@ -81,7 +81,10 @@ def test_read_and_write(self):
                     tr.stats.location = "00"
                     tr.stats.channel = "EHE"
                     tr.stats.calib = 0.199999
-                    tr.stats.delta = 0.005
+                    if format == 'AH':
```

Comment:
Sorry for joining the discussion a bit late - any particular reason to special case this? Why not just do it for all formats? Does it break for some formats?